Part a

sdam12 opened the pull request:
Please review and provide your feedback.Thank You

sdam12 committed:
Part A: Task 1

Re- Structuring of Folders.

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 committed:
Part A: Task 2

Create Make File

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

abhinavachanta reviewed (changes requested):
Please check comments which i added to .cpp files in test folder. Some header file locations could not be resolved.

sdam12 committed:
Changes Done

Thank You for feedback. The Below changes are done
path in the .cpp files modified.

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 committed:
Folders Added

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 commented:
The Changes are taken into consideration. And also updated in the github branch. Please check and let us know if anything else required. Thank You

sdam12 committed:
Task A

Updated Makefile, Sender,Receiver and Subnet main.cpp

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 committed:
Deleted Message.hpp, I already have in /iclude/data_structures/

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 committed:
Update makefile

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

jrajeev459 commented:
The changes made in the make file were good.

cruizm commented on makefile:
```diff
@@ -0,0 +1,29 @@
+	
+clean:
+	rm -f bin/*.exe build/*.o *~
+	-for d in src; do (cd $$d; rm -f *.o *~ ); done
```
This line is not needed

sdam12 replied:
The line is removed from the makefile. Thank You for the feedback.

abhinavachanta commented:
Please update the README file according to the changes in folder structure

sdam12 committed:
Changes in the Readme File and Makefile

Readme File is updated accoding to the folder structure.
Removed line from makefile which professor has requested.

Co-Authored-By: Sagnik Banerjee <nick7695@users.noreply.github.com>

sdam12 commented:
> Please update the README file according to the changes in folder structure

Changes in the Readme.txt file is done. Thank you for feedback.

abhinavachanta commented:
All the requirements look addressed and the codes were executing successfully

abhinavachanta reviewed (approved):
All the requirements look addressed and the codes were executing successfully.

jrajeev459 reviewed (approved):
Everything is working and the .exe were being created and deleted using the commands from make file.
